### PR TITLE
fix(az): add fr-par-3

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

**Comment by ldecarvalho-doc** [comment]
- Added fr-par-3 zone to instances pages.
- Fixed and updated tutorials from the review-doc list

File: compute/instances/api-cli/creating-backups.mdx [hunk 1]
```diff
@@ -30,7 +30,7 @@ POST https://api.scaleway.com/instance/v1/zones/<region>/<uid>/action
 {"action":"backup"}
 ```
 
-Replace `<region>` with the geographical region of the instance. This value can either be `fr-par-1` for instances located in Paris, or `nl-ams-1` for instances located in Amsterdam. Replace `<uid>` with the unique identifier of the instance.
+Replace `<region>` with the geographical region of the instance. This value can be `fr-par-1` for instances located in Paris, for example. Replace `<uid>` with the unique identifier of the instance.
```
Comment (ofranc) [inline compute/instances/api-cli/creating-backups.mdx:33]:
```suggestion
Replace `<region>` with the geographical region of the Instance. This value can be `fr-par-1` for Instances located in Paris, for example. Replace `<uid>` with the unique identifier of the Instance.
```

File: compute/instances/api-cli/migrating-instances.mdx [hunk 2]
```diff
@@ -19,7 +19,7 @@ It is possible to automate the migration between two Instances by using the API.
 
 To migrate an Instance via the API, the instance ID, a valid API Key and the users organization key are required.
 
-1. Start by making a backup of your instance. Replace `<REGION>` with the region of your server, either `fr-par-1` for Instances located in Paris, France or `nl-ams-1` for Instances located in Amsterdam, The Netherlands.
+1. Start by making a backup of your instance. Replace `<REGION>` with the region of your server. This value can be `fr-par-1` for instances located in Paris, for example.
```
Comment (ofranc) [inline compute/instances/api-cli/migrating-instances.mdx:22]:
```suggestion
1. Start by making a backup of your Instance. Replace `<REGION>` with the region of your server. This value can be `fr-par-1` for Instances located in Paris, for example.
```

**Review by ofranc — APPROVED** [review]

File: webhosting/cpanel/reference-content.mdx [hunk 3]
```diff
@@ -2,18 +2,18 @@
...
 dates:
-  validation: 2020-06-29
+  validation: 2021-09-08
   posted: 2021-09-12
 ---
 
 <Message type="note">
 
-This documentation is directed toward all users who dispose of a webhosting package powered by cPanel on [Scaleway Dedibox](https://www.scaleway.com/en/dedibox/cloud-hosting/), and provides useful links and information to manage and configure it.
+This documentation is directed toward all users who dispose of a webhosting package powered by cPanel on [Scaleway Dedibox](https://www.scaleway.com/en/dedibox/cloud-hosting/).
```
Comment (RoRoJ) [inline webhosting/cpanel/reference-content.mdx:11]:
```suggestion
This documentation is for users who have a webhosting package powered by cPanel on [Scaleway Dedibox](https://www.scaleway.com/en/dedibox/cloud-hosting/).
```

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**